sync3/caches/user.go

```diff
@@ -594,11 +500,7 @@ func (c *UserCache) OnSpaceUpdate(ctx context.Context, parentRoomID, childRoomID
 func (c *UserCache) OnNewEvent(ctx context.Context, eventData *EventData) {
 	// add this to our tracked timelines if we have one
 	urd := c.LoadRoomData(eventData.RoomID)
-	if len(urd.Timeline) > 0 {
-		// we're tracking timelines, add this message too
-		urd.Timeline = append(urd.Timeline, eventData.Event)
-		urd.LoadPos = eventData.LatestPos
-	}
+	urd.LoadPos = eventData.LatestPos
```

Comment:
Can we stick a comment above the declaration of LoadPos on the struct? (I keep forgetting what this means). Top of my head:

```
// LoadPos is an event NID. UserRoomData instances represent the status of this room after the corresponding event, as seen by this user.
```